pkg/capabilities/capabilities.go

```diff
@@ -70,7 +70,7 @@ func (c CapabilityType) IsValid() error {
 
 // CapabilityResponse is a struct for the Execute response of a capability.
 type CapabilityResponse struct {
-	Value values.Value
+	Value *values.Map
```

Comment:
why a pointer? concerned we are setting up for nil panics.

Reply:
Because it can be nil, for example when there's an error

Reply:
i don't understand your response. what do you mean `it can be nil`? isn't this change in and of itself changing the type to be "nil"-able for example by either setting nil on error or not setting any value? couldn't you just as a well set an empty value of non-ptr? 

or are you saying that some other part of the system, external to anything changed herein, can set the value to nil? that doesn't make much sense to me since it ought not compile after this type change.

Reply:
Ah, no -- values.Value is an interface, and you can validly set an interface to `nil` (that's what you're doing when you type assert to an interface using the notation `_ values.Value = (*values.Map)(nil)`